src/coincident/plot/matplotlib.py

```diff
@@ -109,3 +207,713 @@ def plot_maxar_browse(item: pystac.Item, overview_level: int = 0) -> plt.Axes:
     da.plot.imshow(rgb="band", add_labels=False, ax=ax)
     ax.set_aspect(aspect=1 / np.cos(np.deg2rad(mid_lat)))
     ax.set_title(item.id)
+
+
+# NOTE: xdem.DEM.hillshade method expects the same x and y resolution
```

Comment:
This gets back to scoping. Should xdem just be a dependency (coincident as 'kitchen sink' package), or instead just focus on search and data organization to hand off to these other libraries?...  In any case, for now I think this basic visual hillshade is convenient to have for plots! The GDAL hillshade implementation is also a bit more sophisticated with handling different x/y/z scales:  https://gdal.org/en/stable/drivers/raster/vrt.html#vrt-that-computes-hillshading-using-an-external-library